packages/headless/src/controllers/facets/category-facet/headless-category-facet.ts

```diff
@@ -83,6 +88,20 @@ export function buildCategoryFacet(
       facetId: getFacetId(),
       ...props.options.facetSearch,
     },
+    executeFacetSearchActionCreator: executeFacetSearch,
+    executeFieldSuggestActionCreator: executeFieldSuggest,
+    select: (value: CategoryFacetSearchResult) => {
+      dispatch(updateFacetOptions());
+      dispatch(
+        executeSearch({
+          legacy: logFacetSelect({
```

Comment:
Why do we dispatch legacy analytics from commerce controllers? Shouldn't we dispatch a noop, or even nothing at all?

Reply:
Ah this actually isn't the commerce category facet controller, nor is it used by it in any way.

It's just that I had to do some refactoring to pass the executeFacetSearchActionCreator, executeFieldSuggestActionCreator, and select as props down to the core category facet and this file was affected.